### PR TITLE
RG351V Theme Updates Part 1, Clean Up, New metadata for Remove ._ files

### PR DESCRIPTION
**Pull request by szalik-rg351**
- [x]  Theme updates for rg351v first part
- [x]  some clean ups of no needed things
- [x]  Update names for rg351p, and put higher community themes ( they just deserve to be higher ) 
- [x]  new metadata for remove ._files, as it was missing i guess
- [x]  Removed handheld simple from basic building correctly

### CODE REVIEW

**Comment by szalik-rg351** [comment]
the metadata update for "Remove ._ files" need to be tested, i copy paste it and replace infromation, so there should not be a problem

**Review by Cebion — APPROVED** [review]
Great Job Szalik

**Review by Cebion — APPROVED** [review]
No Objections

**Review by Cebion — APPROVED** [review]
No Objections

**Review by Cebion — COMMENTED** [review]
Needs to be tested in build

**Review by Cebion — APPROVED** [review]
Approved

File: metadata/RG351P/themes.cfg [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+GBZ https://github.com/szalik-rg351/es-theme-351elec-gbz
+Fundamental https://github.com/szalik-rg351/es-theme-351elec-fundamental
+CosmicRise https://github.com/szalik-rg351/es-theme-351elec-cosmicrise
+SuperSweet https://github.com/szalik-rg351/es-theme-351elec-supersweet
+Minimal https://github.com/szalik-rg351/es-theme-351elec-minimal
+EpicCody https://github.com/szalik-rg351/es-theme-351elec-epic-cody
+Roleta https://github.com/szalik-rg351/es-theme-351elec-roleta
+Supreme https://github.com/szalik-rg351/es-theme-351elec-supreme
+SpinMaster https://github.com/szalik-rg351/es-theme-351elec-spin-master
+Handheld-Simple-NOT-SUPPORTED https://github.com/Rican7/es-theme-handheld-simple
```
Comment (anthonycaccese) [inline metadata/RG351P/themes.cfg:25]: Please change this to just Handheld-Simple.   The theme works well for a set of core systems was one of the first ones that we made for 351ELEC.   Not supported implies that the theme doesn't work at all.

File: metadata/RG351V/themes.cfg [hunk 2]
```diff
@@ -1,25 +1,8 @@
...
-351ELEC-SpinMaster https://github.com/szalik-rg351/es-theme-351elec-spin-master
-Handheld-Simple https://github.com/Rican7/es-theme-handheld-simple
-Alt-Book https://github.com/bebeidon/es-theme-alt-book-3-2
-# This line will be ignored but its needed heheh
+V-Book https://github.com/szalik-rg351/351elec-rg351v-alt-book
+Epic-Cody https://github.com/szalik-rg351/351elec-rg351v-epic-cody
+Minimal https://github.com/szalik-rg351/es-theme-351elec-minimal
+GBZ https://github.com/szalik-rg351/351elec-rg351v-gbz
+Cody-Destroy https://github.com/szalik-rg351/351elec-rg351v-cody-destroy
+Art-Book-Not-Supported-Yet https://github.com/anthonycaccese/es-theme-art-book-3-2
```
Comment (anthonycaccese) [inline metadata/RG351V/themes.cfg:6]:
```suggestion
Art-Book https://github.com/anthonycaccese/es-theme-art-book-3-2
```

Comment (anthonycaccese) [reply metadata/RG351V/themes.cfg:6]: Please remove "Not Supported" from this and Handheld Simple.  The themes work and display ok.

File: metadata/RG351V/themes.cfg [hunk 3]
```diff
@@ -1,25 +1,8 @@
...
+Handheld-Simple-Not-Supported https://github.com/Rican7/es-theme-handheld-simple
```
Comment (anthonycaccese) [inline metadata/RG351V/themes.cfg:7]:
```suggestion
Handheld-Simple https://github.com/Rican7/es-theme-handheld-simple

```

File: metadata/RG351P/themes.cfg (hunk 1 above)
Comment (anthonycaccese) [inline metadata/RG351P/themes.cfg:25]:
```suggestion
Handheld-Simple https://github.com/Rican7/es-theme-handheld-simple

```

File: packages/351elec/config/distribution/modules/gamelist.xml [hunk 4]
```diff
@@ -52,5 +52,18 @@
 		<thumbnail>./downloaded_images/retroarch64-thumb.png</thumbnail>
 		<image>./downloaded_images/retroarch64.png</image>
 	</game>
+	
+	<game>
+		<path>./Remove ._ Files.sh</path>
+		<name>Start Remove ._ Files</name>
```
Comment (anthonycaccese) [inline packages/351elec/config/distribution/modules/gamelist.xml:58]:
```suggestion
		<name>Run Remove ._ Files</name>
```

Comment (anthonycaccese) [reply packages/351elec/config/distribution/modules/gamelist.xml:58]: Would suggest changing this to "Run" in place of "Start"

File: packages/351elec/config/distribution/modules/gamelist.xml [hunk 5]
```diff
@@ -52,5 +52,18 @@
...
+		<desc>Clean up your not needed extensions</desc>
```
Comment (anthonycaccese) [inline packages/351elec/config/distribution/modules/gamelist.xml:59]:
```suggestion
		<desc>Removes hidden ._ files created by the MacOS filesystem</desc>
```

**Review by anthonycaccese — CHANGES_REQUESTED** [review]

**Comment by anthonycaccese** [comment]
Overall this looks great, I made a few suggestions to update before final approval.   Great work, also the new image for clean files looks great.

**Comment by szalik-rg351** [comment]
> Overall this looks great, I made a few suggestions to update before final approval (you should be able to commit each just by pressing the commit suggestion button underneath each above). Great work, also the new image for clean files looks great.

thank you : ) applied

**Review by Cebion — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Review by anthonycaccese — APPROVED** [review]

**Pull request merged**